client/lobbies/lobby.tsx

```diff
@@ -448,7 +258,7 @@ export default class Lobby extends React.Component {
                 isObserver={isObserver}
                 canMakeObserver={!isObserver && canAddObsSlots && team.slots.size > 1}
                 canRemoveObserver={isObserver && canRemoveObsSlots}
-                onAddComputer={onAddComputer && !isLobbyUms ? () => onAddComputer(id) : undefined}
+                onAddComputer={!isLobbyUms && onAddComputer ? () => onAddComputer(id) : undefined}
```

Comment:
You're treating onAddComputer (and all the following callbacks) as optional here but the typings on the props says it's required. If it really *is* optional you should add a `?` to the types